packages/apollo-server-core/src/playground.ts

```diff
@@ -63,7 +63,7 @@ export function createPlaygroundOptions(
       }
     : { settings: undefined };
 
-  const playgroundOptions: PlaygroundRenderPageOptions = {
+  const playgroundOptions: any = {
```

Comment:
Do we want to put the time into type this properly? Seemed to open a can of worms when I tried, as everything that was `{ key1?: { key2: type } }` is now `{ key1?: { key2?: type } }`...